plugins/housekeeping/gsd-disk-space.c

```diff
@@ -998,6 +1033,18 @@ gsd_ldsm_setup (gboolean check_now)
 
         purge_trash_id = g_timeout_add_seconds (3600, ldsm_purge_trash_and_temp, NULL);
         g_source_set_name_by_id (purge_trash_id, "[gnome-settings-daemon] ldsm_purge_trash_and_temp");
+
+        session_bus = g_bus_get_sync (G_BUS_TYPE_SESSION, NULL, NULL);
```

Comment:
`session_bus` is leaked.

Reply:
Ah, no, it’s meant to be like that; it’s a static global which is kept alive for the lifetime of the plugin. A bit pants, but not really a leak.